### PR TITLE
Added Titaniam partner assets

### PR DESCRIPTION
**Pull request by makarthikeyan1**
### Description
Add Titaniam logo and website link to opensearch partners page
 
### Issues Resolved
None

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by stockholmux — CHANGES_REQUESTED** [review]
Just a few tweaks.

File: _partners/titaniam.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: Titaniam Inc.
+# upload your logo to the following directory - must be square
```
[the remark this answers is not in the document]
Comment (makarthikeyan1) [reply _partners/titaniam.md]: done

File: _partners/titaniam.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+logo: '/assets/media/partners/titaniam.png'
```
[the remark this answers is not in the document]
Comment (makarthikeyan1) [reply _partners/titaniam.md:3]: updated. please take a look.

**Comment by stockholmux** [comment]
This looks great, thanks @makarthikeyan1!

**Pull request merged**